bin/composer-require-checker.php

```diff
@@ -17,7 +17,7 @@
     }
 
     $foundAutoloadFile = true;
-    require $autoloadFileLocation;
+    $autoloader = require $autoloadFileLocation;
```

Comment:
Hmm, this is something that shouldn't have been here in first place: it is scary dangerous, as it will mix autoloading definitions of this project and the analysed one.

Also, this effectively evaluates the "files" in a target project: big issue.

@WyriHaximus this is not a flaw in your patch, but in the design of this project

Reply:
Note: similar issues solved in `roave/backwards-compatibility-check` in https://github.com/Roave/BackwardCompatibilityCheck/pull/120 - /cc @maglnet

Reply:
Anyway, it drills down to using the factories in `Roave\BetterReflection\SourceLocator\Type\Composer\Factory\` to look up symbols.

Reply:
@WyriHaximus Maybe we can work together in another patch these suggestions, hm? 👍

Reply:
@carusogabriel sure what do you have in mind? Currently working on using better-reflection for class locating for this PR